cluster-up/cluster/kind-k8s-1.17/README.md

```diff
@@ -0,0 +1,33 @@
+# K8S 1.14.2 in a Kind cluster
+
+Provides a pre-deployed k8s cluster with version 1.14.2 that runs using [kind](https://github.com/kubernetes-sigs/kind) The cluster is completely ephemeral and is recreated on every cluster restart. 
+The KubeVirt containers are built on the local machine and are then pushed to a registry which is exposed at
+`localhost:5000`.
+
+
+## Bringing the cluster up
+
+```bash
+export KUBEVIRT_PROVIDER=kind-k8s-1.14.2
+export KUBEVIRT_NUM_NODES=2 # master + one node
```

Comment:
this not always work on laptops (at least on ipv6)
but i guess we can just leave it for now ?

Reply:
I think if it's not working then I think it should be removed - I personally wasted time on it not working.. @qinqon  - can you check?

Reply:
it works, just not on laptop of some people , this used to be situation
maybe due to firewall but not sure
on modi it works perfect for ipv6 kind, and also mandatory for migration and such

Reply:
Its not work on laptop with RHEL8 and docker installed, docker does not officially support RHEL8.
on laptop with fedora it works, maybe it worth to mention that the host container-runtime should support IPV6

Reply:
i am not sure at all this is the criteria
moreover with this 
https://www.linuxtechi.com/install-docker-ce-centos-8-rhel-8/
(even without cgroups playing i think)
docker-ce does work
i wouldnt add this info imo unless proven,
Miguel and Yuval K for example had problems with 2 nodes on their laptop if i remember right
sometimes it was firewalld (but not sure about it)
also Roman had problems with 2 nodes